doc/interfaces/index.rst

```diff
@@ -10,5 +10,6 @@ DIPY Workflows Interfaces
     data_fetch.rst
     denoise_flow.rst
     registration_flow.rst
+    gibbs_unringing_flow.rst
```

Comment:
This goes before the registration flow file.

Reply:
Addressed in 19657f9